MaxText/train.py

```diff
@@ -146,13 +147,18 @@ def write_metrics_to_tensorboard(writer, metrics, step, config):
 
     full_log = step % config.log_period == 0
 
-    max_logging.log(
-        f"completed step: {step}, seconds: {metrics['scalar']['perf/step_time_seconds']:.3f}, "
-        f"TFLOP/s/device: {metrics['scalar']['perf/per_device_tflops_per_sec']:.3f}, "
-        f"Tokens/s/device: {metrics['scalar']['perf/per_device_tokens_per_sec']:.3f}, "
-        f"loss: {metrics['scalar']['learning/loss']:.3f}"
+    log_message = (
+      f"completed step: {step}, seconds: {metrics['scalar']['perf/step_time_seconds']:.3f}, "
```

Comment:
This may be confusing to users - if each substeptakes 1 second and they have 10 gradient accumulation steps then 10 real seconds went by, but this will only print 1 second - I was thinking to change the message to "seconds per gradient accumulation step" or similar when gradient_accumulation_steps > 1

Reply:
Done